config/key_report_mappings/mappings.py

```diff
@@ -0,0 +1,19 @@
+from collections import defaultdict
+
+from config.key_report_mappings.cof_r2_key_report_mapping import (
+    COF_R2_KEY_REPORT_MAPPING,
+)
+from config.key_report_mappings.cof_r3w2_key_report_mapping import (
+    COF_R3W2_KEY_REPORT_MAPPING,
+)
+
+MAPPINGS = (
+    COF_R2_KEY_REPORT_MAPPING,
```

Comment:
Is this only for COF?

Reply:
Yeah, but theoretically we can make a mapping for others!